zone/tradeskills.cpp

```diff
@@ -468,23 +468,42 @@ void Object::HandleCombine(Client* user, const NewCombine_Struct* in_combine, Ob
 		}
 	}
 
+	//pull the list of components so we can compare during lore check
+	std::string query = fmt::format("SELECT tre.item_id "
```

Comment:
`const auto` maybe? Do we need the alias in the query if you're only referencing a single table and can use the column names without the need for an alias?

Reply:
cleaned up.